core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/AbstractSample.java

```diff
@@ -751,9 +751,10 @@ public Sample build() {
           return SampleImpl.sampleAnalyte(this);
         } else if (sampleTissue != null) {
           return SampleImpl.sampleTissue(this);
+        } else {
+          return SampleImpl.sampleTissueProcessing(this);
         }
       }
-      throw new IllegalArgumentException("No sample can be built with the specified parameters.");
```

Comment:
Can't get rid of this as there are still ways to attempt to create a sample that will fail. Not everything that falls through will be a TissueProcessing sample.